content/docs/user-guide/managing-external-data.md

```diff
@@ -66,21 +66,24 @@ $ dvc run -d data.txt \
 
 ```dvc
 # Add SSH remote to be used as cache location for SSH files
-$ dvc remote add sshcache ssh://user@example.com:/cache
+$ dvc remote add sshcache ssh://user@example.com/abs/path/to/cache
 
 # Tell DVC to use the 'sshcache' remote as SSH cache location
 $ dvc config cache.ssh sshcache
 
 # Add data on SSH directly
-$ dvc add --external ssh://user@example.com:/mydata
+$ dvc add --external ssh://user@example.com/abs/path/to/mydata
 
 # Create the stage with an external SSH output
 $ dvc run -d data.txt \
           --external \
-          -o ssh://user@example.com:/home/shared/data.txt \
-          scp data.txt user@example.com:/home/shared/data.txt
+          -o ssh://user@example.com/abs/path/to/data.txt \
+          scp data.txt user@example.com:/abs/path/to/data.txt
```

Comment:
are you sure that scp will convert it the same way as ssh dependency? (if they have different roots)

Reply:
No, I will have to check ⏳ 

@efiop can you please confirm we want to remove all those `:` from SSH URLs? I think that was a problem on a recent support case and you mentioned we shouldn't have them in docs to avoid confusion.

Reply:
from our urls we can remove `:` , from scp urls - we can't remove `:`

Reply:
> Sorry for the delay, I fell behind again in recent days. Catching up now-Sun.

So I could not confirm this from Windows because while I was able to setup OpenSSH it won't let me private key auth and so I have to type in the password to connect. And apparently this is a limitation of DVC? See iterative/dvc/issues/4392

As a consequence I can't confirm this. @efiop can you please confirm what the text above should be for now? (with or without `:` in the SSH URL)

Reply:
Ah I see Ruslan already confirmed the current text version should be fine!

Reply:
@jorgeorpinel For the record: that issue is not a dvc limitation, but just misconfiguration on your side. Left a comment in that issue.

Reply:
K. Will answer over there.